packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -94,6 +94,23 @@
                     </oc-button>
                   </div>
                 </li>
+                <li
+                  v-for="(mimetype, key) in getMimeTypes()"
```

Comment:
When you moved the `getMimeTypes` getter to the computed props, you'll need to remove the `()` here.

Reply:
Done